build/common/openwisp/tasks.py

```diff
@@ -17,10 +17,12 @@ def radius_tasks():
     management.call_command("delete_old_users",
                             older_than_months=int(os.environ['CRON_DELETE_OLD_USERS']))
 
+
 @shared_task
 def save_snapshot():
     management.call_command("save_snapshot")
 
+
 @shared_task
```

Comment:
I thought that this code could now be added to openwisp-network-topology

Reply:
I'll investigate this ! :smile:

Reply:
If I move celery commands from their own container to the container of the application(example: network-topology container), then the network-topology container would have to run both the celery task and the uwsgi server, which is not inline with the docker philosophy of running one thing in one container, what do you suggest?